packages/react/src/products/hooks/types/useProductsList.ts

```diff
@@ -0,0 +1,30 @@
+import type { Error } from '@farfetch/blackout-client/src/types';
+import type {
+  ListingQuery,
+  SetQuery,
+} from '@farfetch/blackout-client/src/products/types';
+import type {
+  ProductEntity,
+  ProductsListEntity,
+} from '@farfetch/blackout-redux/src/entities/types';
+
+export type ProductsListTypes = {
+  LISTING: 'listing';
+  SET: 'set';
+};
+
+export type UseProductsListParams = {
+  query: ListingQuery | SetQuery;
+  setProductsListHash?: boolean;
+  slug: string;
+  type: ProductsListTypes['LISTING'] | ProductsListTypes['SET'];
+  useCache?: boolean;
+};
+
+export type UseProductsList = {
```

Comment:
```suggestion
export type UseProductsList = (params: UseProductsListParams) => {
```

And this way we simply use one type on our hook. I think this way we keep it consistent with other hooks.

Reply:
Changed :)